core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -193,6 +193,12 @@
           </a>
         </li>
         <hr class="oppia-top-right-menu-item-separator">
+        <li class="nav-item">
+          <a ng-click="$ctrl.closeSubmenuIfNotMobile($event)" href="/contributor-dashboard"
+             class="dropdown-item nav-link protractor-test-contributor-dashboard-link">
+            <span translate="I18N_TOPNAV_CONTRIBUTOR_DASHBOARD"></span>
```

Comment:
Somehow we already have this i18n id in translation .json files: https://github.com/oppia/oppia/blob/develop/assets/i18n/en.json#L577

Reply:
Could we not have this in the navbar yet? I think we should have a period of stabilizing it before we make it generally accessible.